cmake/lzhl.cmake

```diff
@@ -0,0 +1,23 @@
+set(LZHL_DIR ${CMAKE_CURRENT_BINARY_DIR}/_deps/lzhl-src/CompLibHeader)
+
+FetchContent_Populate(lzhl DOWNLOAD_EXTRACT_TIMESTAMP
+    GIT_REPOSITORY https://github.com/xezon/lzhl
```

Comment:
This lzhl needs revisiting and moving to SuperHackers. I copied it from the comment of another fork that used this to make their fork work, but it is unclear if this is the correct version and unedited.

Reply:
The title confirms that it is version 1.0. All good.

The repo has been moved to https://github.com/TheSuperHackers/lzhl-1.0